transforms/packaging/python/pyproject.toml

```diff
@@ -0,0 +1,45 @@
+[project]
+name = "data_prep_toolkit_transforms"
+version = "0.2.1.dev0"
+requires-python = ">=3.10,<3.12"
+keywords = ["transforms", "data preprocessing", "data preparation", "llm", "generative", "ai", "fine-tuning", "llmapps" ]
+description = "Data Preparation Toolkit Transforms"
+license = {text = "Apache-2.0"}
+readme = {file = "README.md", content-type = "text/markdown"}
+authors = [
+    { name = "Maroun Touma", email = "touma@us.ibm.com" },
+]
+
+dependencies = [
+## Code
+	"dpk_code_quality_transform_python    @ git+https://github.com/IBM/data-prep-kit.git#subdirectory=transforms/code/code_quality/python",
```

Comment:
This needs a version tag, maybe.

Reply:
This no longer applies since pypi does not allow @ git+https://...